cypress/integration/main_spec.js

```diff
@@ -0,0 +1,81 @@
+import { O_TRUNC } from "constants";
+import { getFriendlyExpiryDescriptionForHistory } from "../../src/lib/utils";
```

Comment:
I haven't used it, but it looks like the latest version of CRA supports absolute imports which makes imports like this one a little easier.

https://hackernoon.com/absolute-imports-with-create-react-app-4c6cfb66c35d